README.md

```diff
@@ -36,11 +38,11 @@ pipenv run python main.py
 
 [pipenv]: https://pipenv.pypa.io/
 
-
-To connect to Trakt you need to create a new API app: Visit
-`https://trakt.tv/oauth/applications/new`, give it a meaningful name and enter
-`urn:ietf:wg:oauth:2.0:oob`as the redirect url. You can leave Javascript
-origins and the checkboxes blank.
+To connect to Trakt you need to create a new API app:
+1. Visit https://trakt.tv/oauth/applications/new
+1. give it a meaningful name
+1. enter `urn:ietf:wg:oauth:2.0:oob`as the redirect url.
+1. You can leave Javascript origins and the checkboxes blank.
 
```

Comment:
numbers 1, 2, 3, 4
first letter uppercase for all 4 steps

Reply:
no, this is markdown file. this is rendered correctly sequental numbers (see the rendered result). you don't need to manually calculate sequence. I can change to an unordered list if that bothers you.

Reply:
Ok I didn't know that. All good then.

Reply:
Still need approval for a merge.